csrc/root_domain_map.h

```diff
@@ -93,9 +93,22 @@ class TORCH_CUDA_CU_API PairwiseRootDomainMap : public RootDomainMap {
   //! \param is_exact If true, broadcast andnon-broadcast IDs are not mapped
   explicit PairwiseRootDomainMap(
       const TensorView* producer,
-      const TensorView* consumer,
-      bool is_exact = false,
-      bool require_same_extent = true);
```

Comment:
To make option config less confusing, they are now configured with `map***` methods.